providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/e2e/ContainerConfig.java

```diff
@@ -58,6 +69,22 @@ public static GenericContainer flagd(boolean unstable) {
                 .withExposedPorts(8013);
     }
 
+
+    /**
+     * @return a {@link org.testcontainers.containers.GenericContainer} instance of envoy container using
+     * flagd sync service as backend expose on port 9211
+     *
+     */
+    public static GenericContainer envoy() {
+        final String container = "envoyproxy/envoy:v1.31.0";
```

Comment:
@pradeepbbl I have to say I'm quite impressed that you went as far as creating e2e tests for this. Excellent work, very cool.

Reply:
thank you